fullscreen.bs

```diff
@@ -283,14 +289,21 @@ these steps:
 return true if the <a>context object</a> is <a>allowed to use</a> the feature indicated by attribute
 name <code>allowfullscreen</code> and <a>fullscreen is supported</a>, and false otherwise.
 
-<p>The <dfn attribute for=Document><code>fullscreenElement</code></dfn> attribute's getter must
-return <a>context object</a>'s <a>fullscreen element</a>.
-
 <p>The <dfn attribute for=Document><code>fullscreen</code></dfn> attribute's getter must return
 false if <a>context object</a>'s <a>fullscreen element</a> is null, and true otherwise.
 
-<p class=note>Use
-<a attribute for=Document lt=fullscreenElement><code>document.fullscreenElement</code></a> instead.
+<p class=note>Use {{DocumentOrShadowRoot/fullscreenElement}} instead.
+
+<p>The
+<dfn attribute for=DocumentOrShadowRoot id=dom-document-fullscreenelement><code>fullscreenElement</code></dfn>
+attribute's getter must run the following steps and return the result accordingly.
+<ol>
+  <li>If the <a>context object</a> is a <a>shadow root</a> and its <a for=DocumentFragment>host</a>
+  is not <a>connected</a>, return null.</li>
+  <li>Run <a>retargeting</a> <a>fullscreen element</a> against the <a>context object</a>.
```

Comment:
You should store the result in a variable. "Let _candidate_ be the result of ..."